dv/admin.py

```diff
@@ -1,281 +1,171 @@
-#         'status', 'is_tap',
-#     )
-#     list_filter = (
-#         'state', 'status', 'is_tap',
-#     )
-#     search_fields = ('name',)
-#     ordering = ('code',)
-#
-#
-# @admin.register(ProjectTheme)
-# class ProjectThemeAdmin(admin.ModelAdmin):
-#     readonly_fields = ('updated_at',)
-#     list_display = (
-#         'project', 'name',
-#     )
-#     list_filter = (
-#         'name',
-#         'project__financial_mechanism',
-#         'project__priority_sector',
-#         'project__state',
-#         'project__programme_area',
-#     )
-#     search_fields = (
-#         'project__name',
-#     )
-#     ordering = ('project__code',)
-#
-#
-# @admin.register(Project)
-# class ProjectAdmin(admin.ModelAdmin):
-#     readonly_fields = ('updated_at',)
-#     list_display = (
-#         'code', 'name', 'allocation',
-#         'outcome',
-#         'status', 'nuts', 'geotarget',
-#         'has_ended', 'is_dpp', 'is_positive_fx',
-#         'is_improved_knowledge', 'is_continued_coop',
-#     )
-#     list_filter = (
-#         'financial_mechanism',
-#         'priority_sector',
-#         'programme_area',
-#         'state', 'status',
-#         'has_ended', 'is_dpp', 'is_positive_fx',
-#         'is_improved_knowledge', 'is_continued_coop',
-#         'updated_at',
-#     )
-#     search_fields = ('name',)
-#     ordering = ('code',)
-#
-#
-# @admin.register(State)
-# class StateAdmin(admin.ModelAdmin):
-#     readonly_fields = ('updated_at',)
-#     list_display = ('code', 'name', 'url', 'updated_at')
-#     ordering = ('code',)
-#
-#
-# @admin.register(ImportLog)
-# class ImportLogAdmin(admin.ModelAdmin):
-#     readonly_fields = ('created_at', 'data', 'status')
-#     list_display = ('created_at', 'updated_at', 'status')
-#     ordering = ('-created_at',)
-#
-#
-# # just register all the remaining models
-# __models_module = '.'.join(__name__.split('.')[:-1] + ['models'])
-# _models = [m for m in locals().values()
-#            if inspect.isclass(m) and
-#            m.__module__ == __models_module and
-#            issubclass(m, models.Model) and
-#            not (m._meta.abstract or m._meta.proxy)]
-# for model in _models:
-#     try:
-#         admin.site.register(model)
-#     except admin.sites.AlreadyRegistered:
-#         pass
+from django.contrib import admin
+from django.db.models.functions import Length
+from .models import *
+
+
+@admin.register(Allocation)
+class AllocationAdmin(admin.ModelAdmin):
+    list_display = (
+        'state',
+        'programme_area',
+        'financial_mechanism',
+        'gross_allocation',
+    )
+    list_filter = ('state', 'programme_area', 'financial_mechanism', 'funding_period')
+    search_fields = (
+        'state__code', 'state__name',
+        'programme_area__code', 'programme_area__name',
+        'financial_mechanism'
+    )
+
+
+@admin.register(Indicator)
+class IndicatorAdmin(admin.ModelAdmin):
+    list_display = ('id', 'indicator')
```

Comment:
The `Indicator` table has changed its structure pretty much. I would add more fields here: 
`list_display = ('funding_period', 'programme', 'programme_area', 'state', 'indicator', 'header')`
The same thing for `OrganisationRole` which has more fields in the current version.